scss/_variables.scss

```diff
@@ -214,6 +214,7 @@ $enable-grid-classes:                         true !default;
 $enable-pointer-cursor-for-buttons:           true !default;
 $enable-rfs:                                  true !default;
 $enable-validation-icons:                     true !default;
+$enable-negative-margins-utilities:           false !default;
```

Comment:
Is `$enable-negative-margins` enough? Trying to cut down the length of these as for some reason it bothers the hell outta me. 😬

Reply:
It's less clear, but should be fine, especially since we document these variables